scripts/decrypt_config

```diff
@@ -1,13 +1,22 @@
 #!/usr/bin/env bash
 set -e
 
-FILE_IN=app/server/src/resources/config.json.in
-FILE_CLEAR=app/server/src/resources/config.json
+if [ "$#" -eq 1 ]; then
+    FILE_IN_SERVER=app/server/src/resources/config.json.in.$1
+else
+    FILE_IN_SERVER=app/server/src/resources/config.json.in.development
+fi
+
+FILE_CLEAR_SERVER=app/server/src/resources/config.json
 
 export GITHUB_ID=$(vault read -field=clientid secret/beebop/auth/github)
 export GITHUB_SECRET=$(vault read -field=secret secret/beebop/auth/github)
+export PROD_GITHUB_ID=$(vault read -field=clientid secret/beebop/auth/production/github)
+export PROD_GITHUB_SECRET=$(vault read -field=secret secret/beebop/auth/production/github)
+export PROD_GOOGLE_ID=1234
+export PROD_GOOGLE_SECRET=1234
 export GOOGLE_ID=$(vault read -field=clientid secret/beebop/auth/google)
 export GOOGLE_SECRET=$(vault read -field=secret secret/beebop/auth/google)
```

Comment:
this part should also read out the certs from the vault - see /secret/beebop/ssl in the centre vault